packages/cli/bin/cli.js

```diff
@@ -12,64 +12,117 @@ const debug = require('../lib/debug')();
 const minimist = require('minimist');
 const path = require('path');
 const yeoman = require('yeoman-environment');
+const PREFIX = 'loopback4:';
 
-const opts = minimist(process.argv.slice(2), {
-  alias: {
-    version: 'v',
-    commands: 'l',
-  },
-});
-
-if (opts.version) {
-  const ver = require('../package.json').version;
-  console.log('Version: %s', ver);
-  return;
+/**
+ * Parse arguments and run corresponding command
+ * @param env Yeoman env
+ * @param {*} opts Command options
+ * @param log Log function
+ * @param dryRun flag for dryRun (for testing)
+ */
+function runCommand(env, opts, log, dryRun) {
+  const args = opts._;
+  const originalCommand = args.shift();
+  let command = PREFIX + (originalCommand || 'app');
+  const supportedCommands = env.getGeneratorsMeta();
+  if (!(command in supportedCommands)) {
+    command = PREFIX + 'app';
```

Comment:
Should we throw an error saying `command is not supported` or the likes here vs running the default `lb4 app` command?

Reply:
No, we want to allow `lb4 my-app`. Please note `my-app` will be treated as the application name.

Reply:
Sorry, maybe I misunderstood what the code does here. I thought we are silently reverting to `lb4 app` if the command is not supported. I was suggesting to warn/error the user that the command is not supported instead of doing that. I'm okay with `lb4 my-app` since we allow that.